infrastructure/auth.template

```diff
@@ -46,7 +46,7 @@
 					"email"
 				],
 				"VerificationMessageTemplate": {
-					"DefaultEmailOption": "CONFIRM_WITH_LINK",
+					"DefaultEmailOption": "CONFIRM_WITH_CODE",
```

Comment:
Is there any particular reason why confirmation codes are better than confirmation links?

Reply:
I actually would WAY prefer a confirmation link since you can open it on your phone etc and you don't need to type anything in. That being said, if we allow links right now (raw cognito implementation) we need to use the Cognito hosted ui, which I don't want to do. It makes things a bit more confusing in terms of where people land and some other very specific details. 

But given this, we can actually implement our own link based confirmation process that will actually be even nicer I think. We can use a true templated email (HTML, CSS and the works!) that looks a lot more official.